database/insert_e2e.sql

```diff
@@ -0,0 +1,35 @@
+USE `rollcall_test`;
+
+SET foreign_key_checks = 0;
+TRUNCATE lectures;
+TRUNCATE attendance;
+TRUNCATE classes;
+TRUNCATE courses;
+TRUNCATE users;
+SET foreign_key_checks = 1;
+
+INSERT INTO classes (class_id, name) VALUES
+(1, 'SD22w');
+
+INSERT INTO users (user_id, first_name, last_name, email, user_role, password, class_id, date_of_birth) VALUES
+(1, "Teacher1FirstName", "Teacher1LastName", "teacher1@gmail.com", "TEACHER", "$2b$15$uC0sNjstRkK/EBRnCLFkWujCEL.grKhk8NdLSFIccXwVeCR/o6lde", NULL, "1975-10-10"),
```

Comment:
can we maybe just use some proper names for this? :D As everywhere we use normal ones. They could also just match stuff from db if it's hard to come up with new.